RFXtrx/lowlevel.py

```diff
@@ -1610,6 +1615,8 @@ def load_receive(self, data):
         self.raintotal = float((self.raintotal1 << 16) +
                                (self.raintotal2 << 8) +
                                self.raintotal3) / 10
+        if self.subtype == 9:
+            self.raintotal = self.raintotal * 2.54
```

Comment:
The 2.54 scale is only for the value of raintotal3, not for the complete set. So you should be replacing the whole raintotal calculation.

There is also a problem here. Type 9 only contains a flipcounter. It won't be calculating any large values. So it must be accumulated by application late.